apps/react/src/features/studios/components/StudioList/StudioList.tsx

```diff
@@ -0,0 +1,72 @@
+import { memo, FC, useEffect, useRef } from 'react';
+import { List, ListItem, Typography, Button, Paper } from '@mui/material';
+import AddIcon from '@mui/icons-material/Add';
+import { useParams } from 'react-router-dom';
+import { useAppDispatch } from '@js-camp/react/store';
+import { AnimeStudio } from '@js-camp/core/models/studio.model';
+import { setPaginationEvent, updateCursor } from '@js-camp/react/store/studio/slice';
+
+import { StudioListItem } from '../StudioListItem';
+import { StudioSortSelect } from '../StudioSortSelect';
+import { StudioSearch } from '../StudioSearch';
+
+import styles from './StudioList.module.css';
+
+type StudiosListProps = {
+
+	/** Anime studio list. */
+	readonly studios: AnimeStudio[];
+};
+
+const StudiosListComponent: FC<StudiosListProps> = ({ studios }: StudiosListProps) => {
```

Comment:
There is no need to declare type twice:
```suggestion
const StudiosListComponent: FC<StudiosListProps> = ({ studios }) => {
```